src/apps/concepts/components/AddConceptToDictionaryIcon.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import {
+    Fab,
+    Tooltip
+} from "@material-ui/core";
+import {APIConcept} from "../types";
+
+import {Add as AddIcon} from "@material-ui/icons";
+
+
+
+interface AddConceptToDictionaryIconProps {
+    dictionaryToAddTo:string;
+    canAddConcept:string;
+    row:APIConcept|undefined ;
+    addConceptsToDictionary: Function;
```

Comment:
```suggestion
    addConceptsToDictionary: (APIConcept[]) => void;
```

Reply:
@ibacher why we do `addConceptsToDictionary: (APIConcept[]) => void;`

Reply:
Better typing. `Function` is quite generic; `(APIConcept[]) => void` allows me to be certain that anything we're plugging into `addConceptsToDictionary` is a function that takes an array of `APIConcept` objects and returns void. In turn this greatly simplifies the required function.